angular/projects/common/src/app/utils/utils.ts

```diff
@@ -8,15 +12,32 @@ export class Utils {
    * @returns {string} formatted legislation string, or empty string if no legislation values are set.
    * @memberof Utils
    */
-  public static buildLegislationString(obj: Legislation): string {
+  public static buildLegislationString(obj: Legislation, factoryService: any): string {
+    const ACT_CODE_BEGINNING = 'ACT_';
     if (!obj) {
       return '';
     }
 
     const legistrationStrings = [];
-
     if (obj.act) {
-      legistrationStrings.push(obj.act);
+      if (obj.act.substring(0, ACT_CODE_BEGINNING.length) === ACT_CODE_BEGINNING) {
```

Comment:
Again this could be an act util. The comment could be reflected in the name of the function.

Reply:
added an isActCode method and refactored